graylog2-web-interface/src/theme/GraylogThemeContext.jsx

```diff
@@ -13,23 +13,10 @@ function useTheme() {
   return useContext(ThemeColor);
 }
 
-function colorLevel(colorHex, level = 0) {
-  /**
-   * Recreating `color-level` from Bootstrap's SCSS functions
-   * https://github.com/twbs/bootstrap/blob/08ba61e276a6393e8e2b97d56d2feb70a24fe22c/scss/_functions.scss#L97
-   *
-   * @param {string} color - any string that represents a color (ex: "#f00" or "rgb(255, 0, 0)")
-   * @param {number} level - any positive or negative number (negative is brighter)
-   */
-  const colorBase = level > 0 ? teinte.primary.tre : teinte.primary.due;
-  const absLevel = Math.abs(level) * 0.08; // TODO: make 8% a theme variable
-
-  return mix(absLevel, colorBase, colorHex);
-}
-
 const GraylogThemeProvider = ({ children }) => {
   return (
     <ThemeColor.Provider value={defaultValues}>
+      <GlobalThemeStyles />
```

Comment:
Could you explain me your thoughts regarding this change / the related concept? You've implemented this component to provide some global styles, we need for multiple components, right?

Reply:
Yes, this will be where global styles from `public/stylesheets/graylog2.less` will start being migrated. As well as any new globally defined styles we will ned.